app/src/components/modal/modal_deposit_withdraw/index.tsx

```diff
@@ -304,7 +297,9 @@ export const ModalDepositWithdraw = (props: Props) => {
                   setAmountToDisplay('')
                 }}
               />
-              <ModalTitle style={{ marginLeft: '16px' }}>{exchangeType} Asset</ModalTitle>
+              <TYPE.heading3 color={'text1'} margin={'2px 0px 0px 16px'}>
```

Comment:
Also this 2px top margin is making items not be 100% vertically aligned I think its fine having just 16px left margin @pimato wdyt?
![Screenshot 2021-08-30 at 13 25 24](https://user-images.githubusercontent.com/33226956/131332211-a1556558-7935-46cf-bc39-6f3eb8b0537e.png)

Reply:
Yeah that makes sense as long as `Deposit Asset` is vertical centered to the arrow and close icon!

Reply:
Will use other technique than the marginTop for that